src/java.base/share/classes/java/util/TimeZone.java

```diff
@@ -625,15 +625,15 @@ private static TimeZone getTimeZone(String ID, boolean fallback) {
      * both have GMT-07:00, but differ in daylight saving behavior.
      * @see #getRawOffset()
      */
-    public static synchronized String[] getAvailableIDs(int rawOffset) {
+    public static String[] getAvailableIDs(int rawOffset) {
         return ZoneInfo.getAvailableIDs(rawOffset);
```

Comment:
BTW can we call `ZoneInfoFile.getZoneIds` here directly?

Reply:
From OOP point of view - it's strange to call subclass method from a super class.